src/app/components/progress-table/progress-table.component.html

```diff
@@ -170,14 +171,39 @@
         </ion-button>
       </ng-template>
     </ngx-datatable-column>
-  </ngx-datatable>
 
-  <ion-button color="primary" fill="clear" (click)="switchType()">
-    <ng-container *ngIf="type === 'team'">
-      VIEW PROGRESS BY STUDENT
-    </ng-container>
-    <ng-container *ngIf="type === 'student'">
-      VIEW PROGRESS BY PROJECT
-    </ng-container>
-  </ion-button>
+    <ngx-datatable-footer style="position: fixed; bottom: 0px;" *ngIf="true">
+      <ng-template
+        ngx-datatable-footer-template
+        let-rowCount="rowCount"
+        let-pageSize="pageSize"
+        let-selectedCount="selectedCount"
+        let-curPage="curPage"
+        let-offset="offset">
+        <ion-button class="footer-content" color="primary" fill="clear" (click)="switchType()">
+          <ng-container *ngIf="type === 'team'">
+            VIEW PROGRESS BY STUDENT
+          </ng-container>
+          <ng-container *ngIf="type === 'student'">
+            VIEW PROGRESS BY PROJECT
+          </ng-container>
+        </ion-button>
+        <div class="count footer-content">
+          {{ rowCount }} Total
+        </div>
+        <datatable-pager
+          [pagerLeftArrowIcon]="'datatable-icon-left'"
+          [pagerRightArrowIcon]="'datatable-icon-right'"
+          [pagerPreviousIcon]="'datatable-icon-prev'"
+          [pagerNextIcon]="'datatable-icon-skip'"
+          [page]="curPage"
+          [size]="pageSize"
+          [count]="rowCount"
+          [hidden]="!((rowCount / pageSize) > 1)"
+          (change)="page($event)">
```

Comment:
`page($event)` is already specified above in `<ngx-datatable>` either this one here or the one in `<ngx-datatable>` is a redundant

Reply:
i try without `page($event)` in `<datatable-pager>`, then it not calling method have in `<ngx-datatable>`. so i'm remove `page($event)` from `<ngx-datatable>` and keep one in `<datatable-pager>`.